src/vivarium/framework/results/manager.py

```diff
@@ -313,6 +313,30 @@ def register_observation(
     # Helper methods #
     ##################
 
+    @staticmethod
+    def _track_stratifications(
+        measure: str,
+        event_requested_stratification_names: set[str],
+        registered_stratification_names: set[str],
+        missing_stratifications: Dict[str, set[str]],
+        used_stratifications: set[str],
+    ) -> set[str]:
+        """Track unused and missing stratifications for batch-logging"""
```

Comment:
This doc string to me implies this function is going to do something with unused/missing stratifications but you return used stratifications which is a bit confusing

Reply:
@albrja I've updated the comments to hopefully make it clearer?